src/main/java/com/angelbroking/smartapi/smartstream/ticker/SmartStreamTicker.java

```diff
@@ -52,7 +54,7 @@ public class SmartStreamTicker {
      * @throws IllegalArgumentException - if the clientId, feedToken, or SmartStreamListener is null or empty
      */
     public SmartStreamTicker(String clientId, String feedToken, SmartStreamListener smartStreamListener) {
-        if (Utils.isEmpty(clientId) || Utils.isEmpty(feedToken) ||  Utils.validateInputNullCheck(smartStreamListener)) {
+        if (clientId.isEmpty() || feedToken.isEmpty() ||  Utils.validateInputNullCheck(smartStreamListener)) {

```

Comment:
use Strings.isEmpty(string) this takes of null value as well.

Reply:
Changes done.